templates/nginx.template.conf

```diff
@@ -0,0 +1,98 @@
+#user sohamcodes;
```

Comment:
I suspect this file and the other template need some cleanup. For example, they have big commented out sections, and have comments that probably came from elsewhere but may not apply here. And it would be good for the files to have comments at the top describing why they exist/what they're used for.